src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -0,0 +1,153 @@
+#pagebreak()
+= Qualità di Processo 
+
+== Obiettivi 
+
+Si riferisce all'efficacia ed efficienza dei processi e delle metodologie impiegate nello sviluppo, testing e mantenimento del software. Essa abbraccia l'insieme di attività, pratiche e metodi utilizzati lungo l'intero ciclo di vita dello sviluppo del software al fine di garantire che il prodotto finale soddisfi i requisiti specificati e venga consegnato puntualmente e entro il budget previsto. In questo contesto, la qualità di processo è fondamentale per assicurare un'implementazione coerente, misurabile e ottimizzata delle fasi di sviluppo, contribuendo a conseguire gli obiettivi di qualità complessivi.
+
+
+== Processi Primari  
+
+=== Fornitura 
+
+Le metriche di progetto usate nella fornitura sono misure utilizzate per valutare diversi aspetti del processo di sviluppo e del prodotto risultante. Alcune delle metriche comuni utilizzate nella fornitura di progetti software includono:
+
+  - *TCP*: _Total Cost Project_  indica la somma totale dei costi associati al progetto;
+  - *BAC*: _Budget At Completion_  indica il budget totale pianificato per il completamento del progetto;
+  - *EV*: _Earned Value_  rappresenta il valore prodotto dal progetto ossia il valore dei _deliverable_ rilasciati fino al momento della misurazione in seguito alle attività    svolte;
+      - Formula:  $"EV" = "Percentuale di completamento del lavoro" dot "BAC"$.
+  - *PV*: _Planned Value_ Il valore del lavoro pianificato fino a un dato momento;
+        - Formula: $"PV" = "Percentuale di pianificazione del lavoro" dot "BAC"$.
+  - *AC*: _Actual Cost_ Il costo effettivo sostenuto fino a un dato momento;
+  - *CPI*:_Cost Performance Index_ Misura l'efficienza del costo del lavoro svolto fino a un dato momento;
+          - Formula: $"CPI" = "EV" / "AC"$.
+  - *SPI*: _Schedule Performance Index_ Misura l'efficienza del tempo rispetto alla pianificazione;
+        - Formula: $"SPI" = "EV" / "PV"$.
+  - *EAC*: _Estimated at Completion_ Stima del costo totale previsto per completare il progetto;
+        - Formula: $"EAC" = "BAC" / "CPI"$.
+  - *ETC*: _Estimated To Completion_ Stima del costo aggiuntivo necessario per completare il progetto;
+        - Formula: $"ETC" = "EAC" - "AC"$.
+  - *VAC*: _Variance at Completion_ La differenza tra il budget previsto e il budget attuale alla fine del progetto;
+        - Formula: $"VAC" = "BAC" - "EAC"$.
+  - *SV*:_Schedule Variance_ Indica se si è in linea, in anticipo o in ritardo rispetto alla schedulazione delle attività di progetto pianificate nella baseline;
+        - Formula: $"SV" = "EV" - "PV"$.
+  - *CV*:_Cost Variance_ La differenza tra il valore del lavoro effettivamente svolto e il costo effettivo del lavoro svolto fino a un dato momento;
+        - Formula: $ "CV" = "EV" - "AC"$.
+  - *BV*:_Budget Variance_ Indica se alla data corrente si è speso di più o di meno rispetto a quanto previsto a budget alla data corrente;
+        - Formula: $"BV" = "PV" - "AC"$.
```

Comment:
- manca lo spazio tra abbreviazione e termine completo in corsivo;
- a quale _baseline_ ti riferisci?
- per BV direi "..si è speso più o di meno rispetto a quanto inizialmente previsto nel budget." con punto alla fine visto che e' l'ultimo punto.